setup.sh

```diff
@@ -58,19 +58,27 @@ else
 fi
 echo ""
 
+echo "ℹ️  Resolve Swift Packages for Scripts..."
+xcrun --sdk macosx swift package --package-path scripts resolve
+echo "" 
+
 echo "ℹ️  Installing ImageMagick..."
-if [[ -z "${CI-}" ]]; then # skip cache bootstrap for CI
-    echo "Skipping ImageMagick install because not running on CI"
-else
+if [[ -n "${CI-}" ]]; then
```

Comment:
@caldrian suggested to use always `-n` or `-z` to make the script easier to read, so the blocks got inverted 👍